common/config/babel/.babelrc.js

```diff
@@ -2,7 +2,27 @@ const plugins = [];
 process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
   plugins.push([
     '../../common/scripts/babel-conditional-preprocess',
-    { annotations: [{ match: '@conditional-compile-remove-from(stable)' }] }
+    {
+      // Deprecated: Use `features` instead.
+      match: '@conditional-compile-remove-from(stable)',
+      // A list of features recognized by the conditional compilation preprocessing plugin.
+      // "demo" is a special feature, used for demo purposes. For this feature,
+      // The plugin removes any AST node that is preceded by a comment that contains with the tag:
+      // @conditional-compile-remove(demo)
```

Comment:
So we are changing from @conditional-compile-remove-from(stable) to @conditional-compile-remove(stable)
 Haha, a lot of find&replace need to be done!